README.md

```diff
@@ -124,7 +124,18 @@ security:
                 authenticator: contao.security.authenticator
 ```
 
-Edit your `app/config/config.yml` file and add the following:
+Edit your `app/config/config.yml` file:
+
+```yml
+parameters:
+    prepend_locale: false # Set to true if you like, just has to be set
```

Comment:
Shouldn't the setting be added to the `parameters.yml.dist` file?

Reply:
Yes, that makes even more sense if it's a mandatory parameter.

Reply:
It's not mandatory. But I prefer to set it in `app/config/config.yml` via
```yml
contao:
    prepend_locale: true
```
because usually I want this to be the same in all environments of the site.

Reply:
Ok, then it's ok to put it like this into the config.yml. The issue at the moment is, that it's using a parameter that is not defined. We could also do it with a fixed value and without a parameters entry. If nobody has any issues with this, @fritzmg will you adapt the pull request?

Reply:
The parameter is already documented in the README:
```yml
# Contao configuration
contao:
    # Required parameters
    prepend_locale: "%prepend_locale%"
```
This means, that you can either override the config directly in your `config.yml` or use a parameter called `prepend_locale` in your `parameters.yml`.

Reply:
Yes, but this parameter is not in the `parameters.yml.dist` file. So if you just add the package, the build will just crash with a missing parameter which is not really best practice. So we ether add it to the `parameters.yml.dist` or add a static one into the `config.yml`. I don't really care which way, but an error when adding the core-bundle as a symfony bundle is not nice at all : )

Reply:
I see what you mean now. Then I agree it should be added to the `parameters.yml.dist`.

However I still disagree that this configuration should be a parameter in the first place. What is the use case where you want this to be a parameter of the environment?

Reply:
Yes you're right. Lets remove the parameter and make it a static setting. @leofeyer Do you know why this is a parameter in the first place?

Reply:
It was in the `parameters.yml.dist` file of the standard edition, so the Incenteev parameters handler would ask you during install. But since the standard edition is deprecated, we should remove the parameter.

Reply:
@aschempp /cc

Reply:
See https://github.com/contao/manager-bundle/pull/76

Reply:
@leofeyer Does this pull request effect the core-bundle?

Reply:
Only the `README.md` file.